assets/src/components/nav/bottomNavMobile.tsx

```diff
@@ -66,6 +66,7 @@ const BottomNavMobile: React.FC<Props> = ({
             className="m-bottom-nav-mobile__button"
             onClick={() => {
               tagManagerEvent("swings_view_toggled")
+              window.FS?.event("User opened Swings View")
```

Comment:
question (non-blocking): Is it possible to repeatedly click this button, similar to the ViewToggle in leftNav?